### ISSUE TITLE
Add "Personal Health" section

### ISSUE THREAD

**Issue by jonaharagon** [privacyguides/privacyguides.org#1562]
We should look at a section covering personal health/fitness tracking and management, including period tracking as mentioned below, as well as just a generalized health database. Outside of fitness tracking, health databases have a clear use-case for a variety of medical reasons such as medication, vitals, and nutrition tracking; so this definitely isn't something that can be dismissed as simply as "just don't track that stuff if you want privacy" lol

On iOS, simply sticking with Apple Health seems like a clear recommendation, as it operates entirely on-device and includes period tracking as well as tracking of essentially any other health metric. I think we need to research available options on Android still, including the ones linked below:

### Discussed in https://github.com/orgs/privacyguides/discussions/1513

<div type='discussions-op-text'>

<sup>Originally posted by **FriskyInitiate** July  5, 2022</sup>
Period tracker apps are used to record highly personal information that can include your menstruation cycle, sexual activity, daily mood, pain levels, whether you are trying to have a baby, pregnancy or if you experience a miscarriage. With the recent attack on reproductive rights in the US, it seems more relevant than ever to introduce a category for period tracking apps. Consumer Reports has [evaluated](https://web.archive.org/web/20220630092722/https://www.consumerreports.org/health-privacy/what-your-period-tracker-app-knows-about-you-a8701683935/) some of the available apps for privacy invasions and [recommends](https://web.archive.org/web/20220630215118/https://www.consumerreports.org/health-privacy/period-tracker-apps-privacy-a2278134145/) [Drip](https://bloodyhealth.gitlab.io/), [Euki](https://eukiapp.com/) and [Periodical](https://arnowelzel.de/en/projects/periodical).</div>

**Comment by jonaharagon** [comment]
I believe functionality analogous to Apple Health's Health Record functionality is available on Android with [CommonHealth](https://www.commonhealth.org/), which lets you store and access your medical health data locally on your device, but I don't believe it has any personal health tracking capabilities. I think the closest alternatives to Apple Health's HealthKit functionality is found in Google Fit and Samsung Health, neither of which are ideal for our purposes.

On a personal note, I will say that Android's lack of private health tracking that I'm aware of is the primary reason I stick with iOS.

We should be able to find a collection of privacy-respecting individual apps on Android that fulfill different roles Apple Health fulfills on iOS, such as pedometer apps, nutrition apps, period tracking apps, et cetera. They'll merely lack integration with each other or any native integration with Android itself, which should be fine for most people.

https://f-droid.org/en/categories/sports-health/index.html could be promising.

**Comment by freddy-m** [comment]
I don't know of any apps for this, but very much approve of this as a section.

**Comment by devingoodman** [comment]
I have a personal interest in this and have done a lot of research on the matter (primarily on web and iOS -- not android). If I can help turn you on to various apps, software, etc., please just ask.

**Comment by devingoodman** [comment]
I think it would be helpful if you explained the difference between "health data" and "medical data" from a privacy/legal perspective. [Patient-Generated Health Data](https://www.healthit.gov/topic/scientific-initiatives/patient-generated-health-data) (PGHD) is a thing ([definition](https://www.healthit.gov/topic/otherhot-topics/what-are-patient-generated-health-data)) and is not covered by HIPAA. The health data which users voluntarily upload to apps are not private and are not afforded the same protections that medical data is afforded.

It doesn't matter if the data is generated by a wearable or from a doctor's office. If the data in question is provided and voluntarily uploaded by the patient to a 3rd party repository (such as an app), HIPAA protections no longer apply. The only exception to this is if that 3rd party repository is your physician's medical software. If you're uploading PGHD data from your iPhone (Apple Health) via [MyChart](https://apps.apple.com/us/app/id382952264) to your physican's Epic (EMR) instance, then that data is covered under HIPAA. It is covered because it is classified as doctor-patient communications, which is protected- assuming hospital or physician is a [Covered Entity](https://www.hhs.gov/hipaa/for-professionals/covered-entities/index.html). It's important to note that PGHD provided to the physician is not classified as Protected Health Information](https://en.wikipedia.org/wiki/Protected_health_information) (PHI).

tl;dr - 

- Medical Data: Data generated by your doctor or other Covered Entity about you. Covered by HIPAA.
- Health Data: Data generated or provided by you to any app or other 3rd party data repository which isn't your doctor. Not covered by HIPAA.

**Comment by jonaharagon** [comment]
Leaving a note to self: Gadgetbridge for cloudless fitness hardware on Android

**Comment by ph00lt0** [comment]
Do we consider options for apps that would work when forcefully been blocked from having internet access (like possible on grapheneOS)? In that case we only have to look for apps that still work when they are being disconnected.

**Comment by jonaharagon** [comment]
@ph00lt0 Not unless no other options are available in a category, for two reasons: Most users won't be using a ROM which supports blocking internet access from apps entirely, and we want to put an emphasis on apps which are designed with privacy in mind rather than just apps where privacy is (incidentally) possible.

**Comment by Emily-Kang77** [comment]
> we want to put an emphasis on apps which are designed with privacy in mind rather than just apps where privacy is (incidentally) possible.

Then just because a health-tracking app is on F-Droid, it isn't enough to get recommended on the Privacy Guides website? It does make sense, although I think that offering a list of "offline" but not exactly secure apps would help people move away from definite privacy risks like stuff from Google and Apple. Most, or a lot of small open source apps aren't actually checked by people other than the developer, so it is a bit of a gray area. It would be nice if users could directly improve application security with an app like VeraCrypt, but I don't know of any good ones for Android.

**Comment by PrivacyBay** [comment]
The health section of PrivacyTools was released yesterday: https://www.privacytools.io/health

They have added these open-source projects as well:

- https://hovancik.net/stretchly/
- https://workrave.org/
